### PR TITLE
Relayout SQL Editor

### PR DESCRIPTION
**Pull request by xtinec**
- Refactor SQL editor to remove usage of bootstrap col, row and collapse to simplify the layout
- Replace the react-split-pane libraray with react-split to allow custom styling of the gutter area without sacrifice correctness of the ace editor height calculation
- Rewrite the left pane animation via plain css transition and animate it to slide in and out
- General code and css clean up

ezgif com-video-to-gif

### CODE REVIEW

**Comment by kristw** [comment]
LGTM. Will look again when CI completes.

**Comment by xtinec** [comment]
Oops, broke a cypress test. Fixing.

**Review by mistercrunch — APPROVED** [review]
LGTM

File: superset/assets/src/SqlLab/components/SqlEditor.jsx [hunk 1]
```diff
@@ -187,9 +202,44 @@ class SqlEditor extends React.PureComponent {
...
+      <div className="queryPane">
+        <Split
+          sizes={[INITIAL_NORTH_PERCENT, INITIAL_SOUTH_PERCENT]}
+          minSize={200}
+          direction="vertical"
+          gutterSize={GUTTER_HEIGHT}
+          onDragStart={this.onResizeStart}
+          onDragEnd={this.onResizeEnd}
+        >
+          <div ref="north">
```
Comment (kristw) [inline superset/assets/src/SqlLab/components/SqlEditor.jsx]: Should we replace this with `React.createRef()`?

File: superset/assets/src/SqlLab/components/SqlEditor.jsx [hunk 2]
```diff
@@ -187,9 +202,44 @@ class SqlEditor extends React.PureComponent {
...
+              onChange={this.onSqlChanged}
+              queryEditor={this.props.queryEditor}
+              sql={this.props.queryEditor.sql}
+              tables={this.props.tables}
+              height={`${this.state.aceEditorHeight || aceEditorHeight}px`}
+              hotkeys={hotkeys}
+            />
+            {this.renderEditorBottomBar(hotkeys)}
+          </div>
+          <div ref="south">
```
Comment (kristw) [inline superset/assets/src/SqlLab/components/SqlEditor.jsx]: Should we replace this with `React.createRef()`?

File: superset/assets/src/SqlLab/components/SqlEditor.jsx (hunk 1 above)
Comment (xtinec) [reply superset/assets/src/SqlLab/components/SqlEditor.jsx]: Yep, we can.

File: superset/assets/src/SqlLab/components/SqlEditor.jsx (hunk 2 above)
Comment (xtinec) [reply superset/assets/src/SqlLab/components/SqlEditor.jsx]: ditto

**Review by kristw — APPROVED** [review]

**Pull request merged**